app/helpers/lti_configuration_helper.rb

```diff
@@ -0,0 +1,13 @@
+# frozen_string_literal: true
+
+module LtiConfigurationHelper
+  def lms_type_select_options(default_name: "Other Learning Management System")
+    LtiConfiguration.lms_types.collect do |k, v|
+      if k == "other"
```

Comment:
I mentioned this on my other PR, but if we change the value of `other` in the enum to the `default_name` here, we get to remove a lot of special case logic like this.

Reply:
I also left another reply there, but in the #show view specifically we use the functionality of changing the default name from “Other Learning Management System” to “your Learning Management System”.

This view helper is primarily for the drop down in selecting which type of LMS they use, and so I think “Other Learning Management System” makes more sense here, and you’re right, in this case we don’t need to change the default.